server/db/index.js

```diff
@@ -0,0 +1,129 @@
+const Redis = require("ioredis");
+
+const { createGame, deleteGame, getPlayers, addPlayer, getPlayerIndex, playerExists } = require('./game');
+const { getPlayerSocket, setPlayerSocket } = require('./player');
+const {
+    evaluateTrick,
+    getCurrentRound,
+    getCurrentTrick,
+    getPlayerBets,
+    getPlayerCards,
+    getTrickLeader,
+    getTrickWinners,
+    playCard,
+    setCurrentRound,
+    setCurrentTrick,
+    setPlayerBet,
+    startRound,
+    whosTurnIsIt,
+} = require('./round');
+
+const TOTAL_CARDS = 60;
+const MIN_PLAYERS = 3;
+
+
+
+const connect = async () => {
+    const redis = new Redis();
+    try {
+        await redis.connect();
+    } catch (err) { }
+    return redis;
+};
+
+
+const getGameState = async (redis, gameId) => {
```

Comment:
should this be in the game file?

Reply:
no b/c it would cause some circular imports